packages/terra-doc-template/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "html-react-parser": "^0.9.1",
```

Comment:
Is this being used anywhere still? I think this can be removed.

Reply:
Looks like this is still listed as a dependency, we can go ahead and remove it.

Reply:
I think this is pointing to an old commit. This was already removed.